composer.json

```diff
@@ -26,87 +26,6 @@
             "ApplicationTest\\": "module/Application/test/"
         }
     },
-    "extra": {
```

Comment:
This stuff should be preserved :thinking:

Reply:
It's the `laminas-skeleton-installer`, automatically runs on `composer install` and it removes the `extra` portions of the `composer.json`.

Edge case, I did not account for.